src/services/item/plugins/action/base-analytics.test.ts

```diff
@@ -66,5 +79,21 @@ describe('Base Analytics', () => {
     for (const cm of analytics.chatMessages) {
       expect(cm.creator?.createdAt).toBeUndefined();
     }
+
+    const {
+      actions: appActions,
+      data: appData,
+      settings: appSettings,
+    } = Object.values(analytics.apps)[0];
+    for (const aa of appActions) {
+      expect(aa.member?.createdAt).toBeUndefined();
+    }
+    for (const ad of appData) {
+      expect(ad.member?.createdAt).toBeUndefined();
+      expect(ad.creator?.createdAt).toBeUndefined();
+    }
+    for (const as of appSettings) {
+      expect(as.creator?.createdAt).toBeUndefined();
+    }
```

Comment:
Not sure what this is supposed to test ? The non-existence of some member properties ?

Reply:
yes, it was a preventive validation to avoid leaking password. Now it test we don't leak createdAt (that is, we only returns the minimal member). I think I can make it more exhaustive.